### ISSUE TITLE
neural network accuracy being weird

### ISSUE THREAD

**Issue by HinLeung622** [Harry-Westwood/Y4-Project-InterNeuralStellar#4]
@grd349 
I have played around a bit with tensorflow and keras, and did a NN training on a sine wave as instructed, jupyter notebook is here: https://github.com/Harry-Westwood/Fourth-Year-Project/blob/master/Hin's_files/neural_network_test/TestNN.ipynb
The question is, why would both model.fit and model.evaluate both return accuracy = 0 for all epochs even though, at the final plot, the NN got quite a good approximation of the sine function?
